tinacms/tinacms
# Updated generated queries to use pageInfo

Adds pageInfo to generated client

# How to test
- Use a generated query

EX
```ts
const test = await client.queries.postsConnection();
console.log({ pageInfo: test.data.postsConnection.pageInfo });
```
- Verify that that the page info is there.
